pynest/nest/tests/test_spatial/test_spatial_distributions.py

```diff
@@ -131,6 +132,8 @@ def __init__(self, seed, dim, L, N, spatial_distribution, distribution_params=No
                             (math.sqrt(2) * math.log((.1 - 0) / 1))},
             'gaussian': {'p_center': 1., 'sigma': self._L / 4.,
                          'mean': 0., 'c': 0.},
+            'gaussian2d': {'p_center': 1., 'sigma_x': self._L / 4., 'sigma_y': self._L / 4.,
+                           'mean_x': 0., 'mean_y': 0., 'rho': 0., 'c': 0.},
```

Comment:
I think we should test with some other variables than the default ones, in case we for some reason have used the default values instead of parameter names or something similar.

Reply:
No, I was more suggesting using other values in this test, not for `sigma`, but for `mean_x` and `rho` so they are not equal to the default.

Reply:
Done.